R/xml_children.R

```diff
@@ -81,7 +91,7 @@ xml_length.xml_node <- function(x, only_elements = TRUE) {
 #' @export
 xml_length.xml_nodeset <- function(x, only_elements = TRUE) {
   if (length(x) == 0)
-    return(integer)
+    return(integer(0))
```

Comment:
I figured you probably didn't want to return the `integer()` definition. :smile: